### DISCUSSION TITLE
Feature request : support UDP/TCP port fowarding on host without setting up a tun

### DISCUSSION THREAD

**Discussion by ghost** [slackhq/nebula#1011] [General]
Feature request : support UDP/TCP port fowarding without setting up a tun

This will allow nebula to work without root, and one could further set up a socks proxy with this to enable #915

**Comment by johnmaguire** [comment]
I'm not sure I understand the feature request. Could you describe it in more detail?

**Reply by ghost** [reply]
i was looking for something like ngrok which allows you tunnel a port to a remote machine (a local machine can access 127.0.0.1:8080 to talk to, say, 127.0.0.1:8081 on a remote machine without setting up a tun)

**Reply by johnmaguire** [reply]
@nick008a Thanks, I've created #1014 for the feature request.
